apm-sniffer/apm-sdk-plugin/spymemcached-2.x-plugin/src/main/java/org/skywalking/apm/plugin/spymemcached/v2/MemcachedConstructorWithInetSocketAddressListArgInterceptor.java

```diff
@@ -0,0 +1,23 @@
+package org.skywalking.apm.plugin.spymemcached.v2;
+
+import java.net.InetSocketAddress;
+import java.util.List;
+
+import org.skywalking.apm.agent.core.plugin.interceptor.enhance.EnhancedInstance;
+import org.skywalking.apm.agent.core.plugin.interceptor.enhance.InstanceConstructorInterceptor;
+
+public class MemcachedConstructorWithInetSocketAddressListArgInterceptor implements InstanceConstructorInterceptor {
+    
+    @Override
+    public void onConstruct(EnhancedInstance objInst, Object[] allArguments) {
+        StringBuilder memcachConnInfo = new StringBuilder();
+        @SuppressWarnings("unchecked")
+        List<InetSocketAddress> inetSocketAddressList = (List<InetSocketAddress>)allArguments[1];
+        for (InetSocketAddress inetSocketAddress : inetSocketAddressList) {
+            String host = inetSocketAddress.getAddress().getHostAddress();
+            int port = inetSocketAddress.getPort();
+            memcachConnInfo.append(host + ":" + port).append(";");
```

Comment:
You should remove the last `;`.

Reply:
OK!  I have removed it as the following:
 @Override
    public void onConstruct(EnhancedInstance objInst, Object[] allArguments) {
        StringBuilder memcachConnInfo = new StringBuilder();
        @SuppressWarnings("unchecked")
        List<InetSocketAddress> inetSocketAddressList = (List<InetSocketAddress>)allArguments[1];
        for (InetSocketAddress inetSocketAddress : inetSocketAddressList) {
            String host = inetSocketAddress.getAddress().getHostAddress();
            int port = inetSocketAddress.getPort();
            memcachConnInfo.append(host).append(":").append(port).append(";");
        }
        if (memcachConnInfo.length() > 1) {
            memcachConnInfo = new StringBuilder(memcachConnInfo.substring(0, memcachConnInfo.length() - 1));
        }
        objInst.setSkyWalkingDynamicField(memcachConnInfo.toString());
    }